Source/SwiftLintFramework/Rules/LineLengthRule.swift

```diff
@@ -34,24 +36,24 @@ public struct LineLengthRule: ConfigurationProviderRule, SourceKitFreeRule {
             if line.range.length < minValue {
                 return nil
             }
-                        
-            var length = line.content.characters.count
-            
-            // Check if a color literal exists in the line by trying to get range
-            if let colorLiteralRangeStart = line.content.rangeOfString("#colorLiteral("),
-                let colorLiteralRangeEnd = line.content.rangeOfString(")",
+
+            var string = line.content //Mutable copy of content string
+            // While copy of content contains color literal, replace with a single character
+            while string.containsString("#colorLiteral(") {
+                if let rangeStart = string.rangeOfString("#colorLiteral("),
+                let rangeEnd = string.rangeOfString(")",
                     options: .LiteralSearch,
-                    range: colorLiteralRangeStart.startIndex..<line.content.endIndex,
+                    range: rangeStart.startIndex..<string.endIndex,
                     locale: nil) {
```

Comment:
Could you please add indent to lines between `let rangeEnd…` and `locale:nil) {`?